crowbar_framework/app/models/network_service.rb

```diff
@@ -438,4 +439,21 @@ def build_net_info(role, network)
     end
     net_info
   end
+
+  def cidr_to_netmask(cidr, ipv6 = false)
```

Comment:
I'm seeing a lot of this in a number of PR's and gerrit reviews: **ipv6 = false/true**, or **want_ipv6**. I'm wondering if we ought to be thinking about this in terms of ip_version, where we pass around the address family, operate on that, and use it as a suffix for files and things. One may argue this is a nit, but just thinking out loud I wonder if it will keep things more maintainable in the long run.

Reply:
Yeah, that isn't a bad idea. I think ipv6=true/flase and want_ipv6 came from the way these changes came to be. Ie. the pulling the loose thread model :P 
In hindsight address family would make sense.

Reply:
We could merge it for now and refactor the ip version in a follow up. What do you think @rktidwell ?

Reply:
I don't want to stand in the way of progress here, we're following precedent here which is understandable. I'm just putting this out there as a suggestion going forward, no need to block this I suppose.